hw2/03-form.js

```diff
@@ -1,3 +1,34 @@
-/** Exercise 03 - Form **/
+/**
+ * Exercise 03 - Form *
+ *
+ * @format
+ */
 
-// Add your code here
+function displayFormDetails(event) {
+  event.preventDefault();
+  console.log("============Form Submission============");
+
+  // Gettings values of form elements
+  const name = document.querySelector("#name").value;
+  const email = document.querySelector("#contactEmail").value;
+  const feedback = document.querySelector("#contactTextArea").value;
+  const newsletter = document.querySelector("#newsletterCheck");
+
+  //Logging the values of form elements to the console
+  console.log(`   Name: ${name}`);
+  console.log(`   Email: ${email}`);
+
+  // Checking if feedback was submitted
+  if (feedback !== null && feedback !== "") {
```

Comment:
Hi,

Just a question isn't !=null and empty string the same I mean the feedback field would be empty anyways even if it is null or the string is empty, so can't we just have a single condition !=null than the empty string one?

Reply:
Hello Gauri,
Thanks for pointing it out. Actually !=null does not check if the string is empty. But since it is a form field, in case user does not enter anything, it is as good as null and hence in this case can skip the empty check.